apps/web/src/components/Profile/components/PersonalInformation/Display.tsx

```diff
@@ -36,128 +53,197 @@ const Display = ({
 }: DisplayProps) => {
   const intl = useIntl();
   const notProvided = intl.formatMessage(commonMessages.notProvided);
+  const { userAuthInfo } = useAuthorization();
+  const navigate = useNavigate();
+  const routes = useRoutes();
+  const applicationContext = useApplicationContext();
 
-  return (
+  // only show email verification if NOT in an application currently
+  const showEmailVerification = !applicationContext.currentStepOrdinal;
+
```

Comment:
Oh, another case where it is (probably) best to hide it. Viewing an application
Clicking verify triggers verification of the logged in admin 

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/bd092bfa-c965-48ee-90ca-c512a5b8eb48)

Reply:
Might be best to take a whitelist approach to be safest.
So something like 

`const showEmailVerification = isPersonalInformationContext;`

And then additional allowed places can be tagged in like 

`|| isApplicationContext`

Reply:
Shoot, good catch.  I forgot about the admin display.  I like your suggestion of switching to default-off.
[switch to whitelist](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10893/commits/1efeac774d9f383361e439f223a081204a1ec4fe)